### DISCUSSION TITLE
Getting trouble while handling user inputs in my Ruby

### DISCUSSION THREAD

**Discussion by adityachhatre17** [ruby/rubygems#7507] [General]
I have started learning RUBY 2 days ago and  I am stuck at the part of getting user inputs as I tried getting the user input as discussed in the tutorial but its not getting read and showing the output in my terminal. PLEASE HELP!
![Screenshot 2024-03-04 141613](https://github.com/rubygems/rubygems/assets/117889992/529c6e01-2be9-4167-9860-ff9ecb655d99)

**Comment by hsbt** [comment]
@adityachhatre17 This place is not support forum about the Ruby language. I recommend you file it to https://stackoverflow.com/ .
